pack/df.json

```diff
@@ -579,6 +579,24 @@
         "type_code": "operation",
         "uniqueness": false
     },
+    {
+        "code": "26046",
+        "cost": 1,
+        "deck_limit": 3,
+        "faction_code": "jinteki",
+        "faction_cost": 3,
+        "illustrator": "Diana Simonova (Antheia Valour)",
+        "keywords": "Psi - Clone",
+        "pack_code": "df",
+        "position": 46,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "The first time each run the runner approaches this server, you and the Runner secretly spend 0[credit], 1[credit] or 2[credit]. Reveal spent credits. If you and the Runner spent a different number of credits, you may trash this upgrade to force the runner to approach the outermost piece of ice protecting this server.",
+        "title": "Letheia Nisei",
+        "trash_cost": 2,
+        "type_code": "upgrade",
+        "uniqueness": true
+    },    
```

Comment:
the validator is _really_ picky about trailing whitespace. remove the blank spaces after the comma, please.

Reply:
Aha, good to know, thanks! I'll update and see if I can get it through now.